databay/misc/buffers.py

```diff
@@ -0,0 +1,75 @@
+import time
```

Comment:
Might file/class be better placed inside a new `databay/middlewares` directory? Rather than leaving `Buffer` without a home it might be nice to officially create middlewares that we can build on in the future.

Reply:
Good suggestion! Having read your comment I think that we should indeed create another directory, but I'd think of a different name. Dunno maybe `support`? Cause `middlewares` - can we think of any other middleware? Quite frankly I can't, and if I were to venture out into the imagination I'd say we'd maybe put another one or two, but most likely not 10. Sounds like too little for a whole directory just for one file. What do you say?

Reply:
I definitely can't think of any other middlewares right now. I guess the reason I suggested it is it's a kind of common pattern I've seen for injecting optional secondary behaviors into an API but I'm not committed to it in any direction. A support directory would be equally fine.

Reply:
Yupp I'm 100% with you on that this is a common pattern, but I also can't think of any other middleware for Databay, which would make this folder distinction a little confusing, suggesting that there should be more middleware - which we can't anticipate at the moment. Let's go with `support`, great suggestion 👍